source/documentation/partials/_messagetypeintro.md

```diff
@@ -1,4 +1,20 @@
-## Accessing the XML Schemas
+## Message details
+
+This section provides the following information about each message type: 
+
+- Characteristics of the data groups belonging to the message: sequence, number of repetitions, and a status value to indicate if the data group is: 
+    - mandatory (R: Required), 
+    - optional (O: Optional) or 
```

Comment:
What's the guidance on use of the Oxford comma?

Personally, I favour the Oxford comma, so I would prefix "or" with a comma.